poly-commit/src/linear_codes/mod.rs

```diff
@@ -226,14 +226,15 @@ where
     ) -> Result<
         (
             Vec<LabeledCommitment<Self::Commitment>>,
-            Vec<Self::Randomness>,
+            Vec<Self::CommitmentState>,
         ),
         Self::Error,
     >
     where
         P: 'a,
     {
         let mut commitments = Vec::new();
+        let mut states = Vec::new();
```

Comment:
Very minor optimization, but you know how many polynomials you have, so you can use `Vec::with_capacity()` here

Reply:
I see, but do you see a way to access the length of `polynomials`?

Reply:
can't you just get `.len()`?

Reply:
No, because `polynomials: impl IntoIterator<Item = &'a LabeledPolynomial<F, P>>,`